sources/packages/web/src/components/institutions/modals/AddInstitutionUserModal.vue

```diff
@@ -134,6 +154,7 @@ export default {
     const usersList = ref();
     const institutionLocationList = ref();
     const payLoad = ref({} as InstitutionUserAuthDetails);
+    const selectedAdminRole = ref({ name: "admin", code: "admin" } as UserAuth);
```

Comment:
may be default value shouldn't be `admin`. what if when loading time increases and the institution user is not admin

Reply:
This is selectedAdminRole which will not be considered if the API returns user type of anything other than Admin.
In case of API taking time to load, Admin role select will not load at all and also will not be a part of any API call. 
Only intention for the initialization is in admin role dropdown, admin should be the default selected value.